persistence.js

```diff
@@ -1,7 +1,7 @@
 'use strict'
 
 var util = require('util')
-var urlModule = require('url')
+var urlModule = require('native-url')
```

Comment:
Why this change?

Reply:
> 'url.parse' was deprecated since v11.0.0. Use 'url.URL' constructor instead node/no-deprecated-api

Unfortunally `new Url` doesn't work much like `url.parse` so this is a lightweight replacement for it (https://github.com/GoogleChromeLabs/native-url#readme)